libreyolo/models/rfdetr/model.py

```
@@ -176,9 +175,8 @@ class LibreRFDETR(BaseModel):
     FILENAME_PREFIX = "LibreRFDETR"
     INPUT_SIZES = {"n": 384, "s": 512, "m": 576, "l": 704}
     SEG_INPUT_SIZES = {"n": 312, "s": 384, "m": 432, "l": 504, "x": 624, "xx": 768}
-    # Pose uses the dedicated GroupPose preset (adapted from RF-DETR v1.8.0);
-    # the keypoint preview runs the encoder at a 576 square.
-    POSE_INPUT_SIZES = {"x": 576}
+    # Pose checkpoints use the dedicated RFDETR_POSE_CONFIGS resolutions.
+    POSE_INPUT_SIZES = {"n": 384, "s": 512, "m": 576, "l": 704, "x": 576}
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)